### DISCUSSION TITLE
PM2 - Cluster mode can't be run with next start

### DISCUSSION THREAD

**Discussion by conanbatt** [vercel/next.js#41025] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

    Operating System:
      Platform: darwin
      Arch: x64
      Version: Darwin Kernel Version 21.6.0: Mon Aug 22 20:19:52 PDT 2022; root:xnu-8020.140.49~2/RELEASE_ARM64_T6000
    Binaries:
      Node: 18.9.0
      npm: 8.19.1
      Yarn: 1.22.17
      pnpm: N/A
    Relevant packages:
      next: 12.3.2-canary.12
      eslint-config-next: 12.3.1
      react: 18.2.0
      react-dom: 18.2.0

### Describe the Bug

This is a follow-up issue to this one: https://github.com/vercel/next.js/discussions/36606

Running pm2 in cluster mode throws a "pm2.configuration.json/.env" failed to load error (file does not exist)
Pm2 in cluster mode appends the path with the configuration file and it is documented on their website with a nuxt example [here](https://pm2.keymetrics.io/docs/usage/docker-pm2-nodejs/) 

This can be worked around with a custom server as you can change the path/arguments of the processes. 

### Expected Behavior

Expect the application to run.

### Link to reproduction

https://github.com/conanbatt/next-pm2-cluster-issue

### To Reproduce

To reproduce:
1. Fresh npx create-next-app
2. yarn add pm2
3. Add json file:
```
// pm2.json
{
  "apps": [
    {
      "name": "my-app",
      "script": "npm",
      "args": "start",
      "instances": "1",
      "exec_mode": "cluster"
    }
  ]
}

```
4. yarn pm2 start pm2.json

**Comment by ycscholes** [comment]
Please try the solutions below:
1. modify the script command to "node_modules/next/dist/bin/next"
```
"apps": [
   {
      "script": "node_modules/next/dist/bin/next",
    }
  ]
```

2. modify the npm start command in package.json to "next start ./"
```
"scripts": {  
    "start": "next start ./",  
  },
```

Hope it can help you

**Reply by indomitableXiaoNan** [reply]
Useful suggestions，thx
